docker-app/qfieldcloud/core/filters.py

```diff
@@ -0,0 +1,44 @@
+import django_filters
+from django.db import models
+
+from qfieldcloud.core.models import Project, ProjectQueryset
+
+
+class IncludePublicChoices(models.IntegerChoices):
+    EXCLUDE = 0, "Exclude"
+    INCLUDE = 1, "Include"
+
+
+class ProjectFilterSet(django_filters.FilterSet):
+    owner = django_filters.CharFilter(field_name="owner__username", lookup_expr="exact")
```

Comment:
```suggestion
    owner = django_filters.CharFilter(field_name="owner__username", lookup_expr="iexact")
```
Case insensitive search for owner, is it OPENGIS or openGIS :) They are case insestive unique anyways.

Reply:
Done.